Cargo.toml

```diff
@@ -14,14 +14,13 @@ categories = ["no-std", "data-structures"]
 keywords = ["generic-array"]
 readme = "README.md"
 edition = "2021"
-rust-version = "1.65"
+rust-version = "1.81.0"
```

Comment:
I think it's better to use just `1.81` here.

Reply:
It's the same logic as with skipping zero minor/patch versions in dependencies (e.g. we use `zeroize = "1.8"`, not `zeroize = "1.8.0"`). The notations mean the same thing, but `1.81` is shorter and easier to read. I think it can be updated in a pre-release PR.